src/Repository/ElectionRoundRepository.php

```diff
@@ -40,4 +40,21 @@ public function getAllRoundsAsChoices(): array
 
         return $rounds ?? [];
     }
+
+    public function getUpcomingElectionRounds(): array
+    {
+        $results = $this->createQueryBuilder('r')
```

Comment:
`$this->createQueryBuilder('r', 'r.id')` and don't need the `foreach`, the result array will indexed by id
but each item of array will contents two columns: id and label